docs/src/examples/scikit-learn.rst

```diff
@@ -0,0 +1,112 @@
+********************
+Scikit-learn
+********************
+.. Might also think of moving this file to examples/ then we an example auto contained in the
+   repository. We invoke this file from index.rst
+
+In this example, we're going to demonstrate how Oríon can be integrated to a minimal model using
+scikit-learn [scikit]_ on the iris dataset [iris]_. The files mentioned in this example are
+available in the folder :file:`examples/scikitlearn-iris/`.
+
+The requirements are listed in requirements.txt. You can quickly install them using ``$ pip
+install -r requirements.txt``. If you haven't installed Oríon previously, make sure to
+:doc:`configure it properly <../install/core>` before going further.
+
+Sample script
+---------------
+
+.. literalinclude:: ../../../examples/scikitlearn-iris/main.py
+   :language: python
+   :lines: 1-2, 5-9, 13-30
+
+This very basic script takes in parameter one positional argument for the hyper-parameter *epsilon*
+which control the loss in the script.
+
+The script is divided in five parts:
+
+#. Parsing of the script arguments
+#. Loading and splitting the dataset
+#. Training a classifier using the researcher-defined *epsilon*
+#. Evaluating the classifier using the testing set
+#. Reporting the performance of the model. i.e., the accuracy.
+
+.. note::
+   The workflow presented in the script is simplified compared to the reality on purpose. The
+   objective of this example is to illustrate the basic steps involved in using Oríon.
+
+To find a good *epsilon*, a user would run empirically ``$ python main.py <epsilon>`` multiple
+times, choosing a new value for *epsilon* manually.
+
+This ad-hoc hyper-parameter optimization is unreliable, slow, and requires a lot of work from the
+user. Oríon solves this problem by providing cutting-edge hyper-parameter optimization
```

Comment:
```suggestion
user. Oríon solves this problem by providing established hyper-parameter optimization
```
We prioritize established and reliable algorithms.

Reply:
Good catch! Thanks!